### PR TITLE
Adding handover document to FP Syllabus

### PR DESCRIPTION
**Pull request by kfklein15**
Signed-off-by: KFK <80272258+kfklein15@users.noreply.github.com>

## What does this change?

Module: Final Projects
Week(s): 4

## Checklist

- [x] I have read the [contributing guidelines](https://syllabus.codeyourfuture.io/contributing/overview)
- [x] I have checked my spelling and grammar with an [automated tool](https://www.grammarly.com/grammar-check)
- [x] I have previewed my changes to check the [markdown renders](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) as I intend
- [x] I have run my code to check it works
- [x] My changes follow our [Style Guide](https://syllabus.codeyourfuture.io/guides/code-style-guide)

## Description

Adding the Handover document link and what should be done so we don't forget it

## Who needs to know about this?

 @martaesn @AliciaE08 @ShaunCYFSA @haldenb

### CODE REVIEW

File: docs/finalproject/week-4/lesson.md [hunk 1]
```diff
@@ -7,3 +7,5 @@ sidebar_label: Lesson
 See [Week 2](../week-2/lesson) for instructions for this week. 
 
 Make sure to check in again on the contributions per team members, as detailed on [Week 3](../week-3/lesson)
+
+The team should be asked to fill out the [handover document](https://docs.google.com/document/d/1hWrWS1kyHfJnzSeTOc7mwjElSJx_xtk6Down1Bzlaos/edit) once they have finished the final project development. This document is important to ensure the Product Owner and technical team that would continue working on this product has all necessary information about what has been done.
```
Comment (SallyMcGrath) [inline docs/finalproject/week-4/lesson.md]:
```suggestion
Each team must fill out their [handover document](https://docs.google.com/document/d/1hWrWS1kyHfJnzSeTOc7mwjElSJx_xtk6Down1Bzlaos/edit) once they have finished the final project development. This document is important to ensure the Product Owner and technical team that would continue working on this product has all necessary information about what has been done.
```

Comment (SallyMcGrath) [reply docs/finalproject/week-4/lesson.md]: (Active language please)

**Review by SallyMcGrath — DISMISSED** [review]
Have asked for a tweak that you can accept right on the line change.

The trainees would massively benefit from identifying around three quantified impact statements they can make about their project. This will really really help their CV and interview performance. Can you find a way to include it in this document or in this handover process?

File: docs/finalproject/week-4/lesson.md (hunk 1 above)
Comment (kfklein15) [reply docs/finalproject/week-4/lesson.md]: (Excellent point, thanks! It's the automatic translation in my head)
I have updated the handover document and tagged you.

**Review by esma-g — APPROVED** [review]
The link is not accessible without CYF email.

**Review by illicitonion — APPROVED** [review]

**Pull request merged**